src/invoices/InvoicesList/Invoice/InvoiceEditLayer.js

```diff
@@ -22,10 +25,26 @@ class InvoiceEditLayer extends Component {
       path: `${INVOICE_API}/:{id}`,
       throwErrors: false,
     },
+    user: {
+      ...USER,
+      GET: {
```

Comment:
why not to have it in resources as `approvedByUser` resource or smth like that, copy paste

Reply:
fixed